src/renderer/common/analytics.ts

```diff
@@ -1,30 +1,33 @@
 import amplitude from 'amplitude-js';
-
-import { useAppSelector } from '../hooks';
-import { selectConfigState, ConfigKey } from '../data/Config/configState';
+import store from 'renderer/store';
+import { ConfigKey } from '../data/Config/configState';
 
 const AMPLITUDE_KEY = 'f1cde3642f7e0f483afbb7ac15ae8277';
 const AMPLITUDE_HEARTBEAT_INTERVAL = 3600000;
 
 amplitude.getInstance().init(AMPLITUDE_KEY);
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-const Analytics = (event: string, metadata: any) => {
-  const storedConfig = useAppSelector(selectConfigState);
+const analytics = (event: string, metadata: any) => {
+  const storedConfig = store.getState().config.values;
   if (storedConfig) {
-    if (
-      process.env.NODE_ENV !== 'development' &&
-      ConfigKey.AnalyticsEnabled in storedConfig &&
-      storedConfig[ConfigKey.AnalyticsEnabled] === 'true'
-    ) {
-      amplitude.getInstance().logEvent(event, metadata);
+    if (process.env.NODE_ENV !== 'development') {
+      if (
+        ConfigKey.AnalyticsEnabled in storedConfig &&
+        storedConfig[ConfigKey.AnalyticsEnabled] === 'true'
+      ) {
+        amplitude.getInstance().logEvent(event, metadata);
+      }
+    } else {
+      // eslint-disable-next-line no-console
+      console.log('analytics event', event);
     }
   }
 };
 
-// Analytics('openApp', {});  // TODO: disabled this one as it crashes sometimes
+analytics('openApp', {}); // TODO: disabled this one as it crashes sometimes
```

Comment:
I guess a bit of testing to make sure the TODO is removable :)

Reply:
do you remember if it was a hooks related warning? it didn't like any `analytics` call until I took that `useAppSelector` out of it

Reply:
I am going to say - its plausible